### ISSUE TITLE
`no-implicit-any-catch` fixer produces invalid code for `catchError`

### ISSUE THREAD

**Issue by JasonWeinzierl** [JasonWeinzierl/eslint-plugin-rxjs-x#224]
When adding `: unknown` to the `error` callback argument of the following example, invalid code is produced:

```ts
import { throwError, catchError } from 'rxjs';

throwError(new Error('Kaboom!')).pipe(
  catchError((error, caught) => console.error(error)),
);
```

Invalid output:

```ts
import { throwError, catchError } from 'rxjs';

throwError(new Error('Kaboom!')).pipe(
  catchError(((error: unknown), caught) => console.error(error)),
);
```

This was reported and fixed in the smarttools fork: https://github.com/DaveMBush/eslint-plugin-rxjs/issues/176
